components/QrScanner.tsx

```diff
@@ -16,117 +22,148 @@ import {SvgImage} from './ui/svg';
+                margin="0 0 5 0"
+                style={Theme.CameraDisabledStyles.bannerTitle}>
+                {t('cameraAccessDisabled')}
+              </Text>
+              <Text
+                testID="cameraPermissionGuide"
+                color={Theme.Colors.whiteText}
+                style={Theme.CameraDisabledStyles.bannerGuide}>
+                {t('cameraPermissionGuideLabel')}
+              </Text>
+            </Column>
+            <Pressable>
+              <Icon
+                testID="close"
+                name="close"
+                onPress={() => setShowCameraPermissionDeniedBanner(false)}
+                color={Theme.Colors.whiteText}
+                size={18}
+              />
+            </Pressable>
+          </Row>
+          <Row
+            style={Theme.CameraDisabledStyles.bannerEnablePermissionContainer}>
+            <Pressable
+              onPress={openSettings}
+              style={Theme.CameraDisabledStyles.bannerEnablePermission}>
+              <Text
+                testID="EnablePermissionText"
+                color={Theme.Colors.whiteText}>
+                {t('EnablePermission')}
+              </Text>
+            </Pressable>
+          </Row>
+        </Column>
       </View>
     );
   };
   return (
-    <Column fill align="space-between" margin="0 0 60 0">
-      <View style={Theme.Styles.scannerContainer}>
-        {hasPermission ? (
-          <Camera
-            {...testIDProps('camera')}
-            style={Theme.Styles.scanner}
-            barCodeScannerSettings={{
-              barcodeTypes: [BarCodeScanner.Constants.BarCodeType.qr],
-            }}
-            onBarCodeScanned={scanned ? undefined : onBarcodeScanned}
-            type={cameraType}
-          />
-        ) : (
-          <View style={Theme.Styles.disabledScannerContainer} />
-        )}
-      </View>
-      {props.title && (
-        <Text
-          testID="holdPhoneSteadyMessage"
-          align="center"
-          weight="semibold"
-          style={Theme.TextStyles.base}
-          margin="0 57">
-          {props.title}
-        </Text>
+    <>
+      {hasCameraPermission ? (
+        <Column style={Theme.CameraEnabledStyles.container}>
+          <View style={Theme.CameraEnabledStyles.scannerContainer}>
+            <Camera
+              {...testIDProps('camera')}
+              style={Theme.CameraEnabledStyles.scanner}
+              barCodeScannerSettings={{
+                barcodeTypes: [BarCodeScanner.Constants.BarCodeType.qr],
+              }}
+              onBarCodeScanned={scanned ? undefined : onBarcodeScanned}
+              type={cameraType}
+            />
+          </View>
+          <Column fill align="space-between" style={{marginTop: 24}}>
+            {props.title && (
+              <Text
+                testID="holdPhoneSteadyMessage"
+                align="center"
+                style={Theme.CameraEnabledStyles.holdPhoneSteadyText}
+                margin="0 57">
```

Comment:
Should we move these to styles?

Reply:
Custom text component is accepting certain props and it is applying theme spacing and margin internally inside the component. So i think it would be good to pass these as separate prop to text component and we can change the format when refactoring this component.